source/base/ReferenceCounter.ooc

```diff
@@ -22,6 +21,12 @@ ReferenceCounter: class {
 		this _lock = Mutex new(mutexType)
 //		"  RC, #{This count}, #{this _target class name}" println()
 	}
+	free: override func {
+		This count -= 1
+//		"--RC, #{This count}, #{this _target class name}" println()
```

Comment:
Fixed. I might as well do some minor cleanup while i'm at it. I'll only correct the most obvious and bothersome style violations though. We'll have to make another pass at a later date.